cmd/create/network/cmd.go

```diff
@@ -0,0 +1,157 @@
+package network
+
+import (
+	"context"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"gopkg.in/yaml.v3"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/interactive"
+	"github.com/openshift/rosa/pkg/network"
+	helper "github.com/openshift/rosa/pkg/network"
+	"github.com/openshift/rosa/pkg/ocm"
+	bsOpts "github.com/openshift/rosa/pkg/options/network"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+type NetworkSpec struct {
+	Service network.NetworkService
+}
+
+type networkStruct struct {
```

Comment:
Is this struct needed? It looks like it's boxing the "service" without any additional methods or functionality attached - could we just use the service directly?